packages/@coorpacademy-nova-icons/scripts/generate-icons.js

```diff
@@ -41,8 +41,7 @@ type JSXElement = {|
   attributes?: Array<JSXAttribute>
 |};
 
-const isFillAttribute = ({name: {name}, value: {value} = {}}: JSXAttribute): boolean =>
-  name === 'fill' && value !== 'none';
+const isFillAttribute = ({name: {name}, value: {value} = {}}: JSXAttribute): boolean => name === 'fill' && value === '#757575';
```

Comment:
avant le generate-icons ne gardé pas les autres couleurs, maintenant il remplace #757575 par currentColor et le reste il les laisse -as is- pour plus de simplicité